_studio/shared/umc/codec/mpeg2_dec/src/umc_mpeg2_decoder.cpp

```diff
@@ -794,6 +792,7 @@ namespace UMC_MPEG2_DECODER
         }
 
         frame.currFieldIndex = frame.GetNumberByParity(picExt.picture_structure == BOTTOM_FLD_PICTURE);
+        frame.fieldIndex = frame.currFieldIndex;
```

Comment:
Need explanation what currFieldIndex and fieldIndex. What the difference is.

Reply:
Thanks for your good suggestion, code updated, pls help review it again.